IyiKuyoro/Pals
# #161534306: Setup eslint with airbnb

#### What does this PR do?
Setup eslint

#### How should this be manually tested?
>- Clone the repository to your local machine
>- Open the scripts and use the linter to check for bugs

#### What are the relevant pivotal tracker stories?

#161534306
